packages/jest-runtime/src/__tests__/transform-test.js

```diff
@@ -118,27 +113,23 @@ describe('transform', () => {
     const response2 = transform('/fruits/banana.js', config);
     expect(response2).toBe(response);
 
-    transform('/fruits/kiwi.js', config, {
-      instrument: () => 'instrumented kiwi',
-    });
+    transform('/fruits/kiwi.js', config);
     const snapshot = vm.Script.mock.calls[1][0];
     expect(snapshot).toMatchSnapshot();
 
-    // reset and make sure `instrument` returns a different value.
-    jest.resetModuleRegistry();
-    reset();
-    config.collectCoverage = true;
+    transform(
+      '/fruits/kiwi.js',
+      Object.assign({}, config, {collectCoverage: true}),
+    );
 
-    transform('/fruits/kiwi.js', config, {
-      instrument: null,
-    });
     expect(vm.Script.mock.calls[0][0]).not.toEqual(snapshot);
     expect(vm.Script.mock.calls[0][0]).not.toMatch(/instrumented kiwi/);
 
     // If we instrument again, we get a different result.
```

Comment:
This comment is outdated with your change. "If we disable coverage, we get a different result."